test/meter/meter-radial.visual-diff.js

```diff
@@ -0,0 +1,35 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-meter-radial', function() {
+
+	const visualDiff = new VisualDiff('meter-radial', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/test/meter/meter-radial.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	it('no-progress', async function() {
+		const rect = await visualDiff.getRect(page, '#no-progress');
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+	});
+
+	it('has-progress', async function() {
+		const rect = await visualDiff.getRect(page, '#has-progress');
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+	});
+
+	it('completed', async function() {
+		const rect = await visualDiff.getRect(page, '#completed');
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+	});
```

Comment:
Nit: if all the tests are the same, I found using a `forEach` over an array handy. Have a look at the icon tests for an example.

Reply:
I'll similarly tidy up the others in buttons & typography when I get a chance.